src/schedule/task.c

```diff
@@ -69,7 +69,8 @@ void task_main_init(void)
 	struct task_ops ops = {
 		.run = main_main,
 		.get_deadline = task_main_deadline,
-		.complete = NULL
+		.complete = NULL,
+		.is_ready = NULL
```

Comment:
again - using a comma also in the last line helps. Also, there's no need to assign NULL to struct members during initialisation - that's done automatically. Can be removed throughout the patch set

Reply:
Done.